.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,90 @@
+name: benchmarks
+
+on:
+  schedule:
+    - cron:  '0 3 * * *'
+  workflow_dispatch:
+
+jobs:
+  benchmarks:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Unshallow
+        run: git fetch --prune --unshallow
+      -
+        name: Read go version
+        id: go-version
+        shell: bash
+        run: |
+          content=`cat ./.go-version`
+          echo "::set-output name=content::$content"
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          # TODO: Replace with go-version-from-file when it is supported
+          # https://github.com/actions/setup-go/pull/62
+          go-version: ${{ steps.go-version.outputs.content }}
+      -
+        name: Go mod download
+        run: go mod download -x
+      -
+        name: Run benchmarks
+        id: bench
+        run: |
+          go test ./internal/langserver/handlers \
+            -bench=InitializeFolder_basic \
+            -run=^# \
+            -benchtime=60s \
+            -timeout=30m | tee ${{ runner.temp }}/benchmarks.txt
+      -
+        name: Evaluate benchmarks
+        id: bench-eval
+        run: |
+          BENCHDATA="go run go.bobheadxi.dev/gobenchdata"
+
+          BENCH_PATH="${{ runner.temp }}/benchmarks.txt"
+          DATA_PATH="${{ runner.temp }}/benchdata.json"
+          RESULTS_PATH="${{ runner.temp }}/benchdata-results.json"
+          CHECKS_CONFIG_PATH="${{ github.workspace }}/.github/gobenchdata-checks.yml"
+
+          cat $BENCH_PATH | $BENCHDATA --json ${DATA_PATH} -v "${GITHUB_SHA}" -t "ref=${GITHUB_REF}"
+
+          $BENCHDATA checks eval \
+            ${DATA_PATH} \
+            ${DATA_PATH} \
+            --checks.config ${CHECKS_CONFIG_PATH} \
+            2> $RESULTS_PATH # eval returns result JSON to stderr for some reason
```

Comment:
Ah this command is intended to be used with the `--json [path]` flag, and probably should not print JSON unless that flag is provided - I'm fixing this with https://github.com/bobheadxi/gobenchdata/pull/53 ! (though that would be a breaking change here)